src/OrbitGgp/InstanceTests.cpp

```diff
@@ -93,8 +94,8 @@ TEST(InstanceTests, GetListFromJson) {
                           "key\":\"other value\",\"other complex object\":{\"object "
                           "key\":\"object value\"}}]")
                           .toUtf8();
-    const auto result = Instance::GetListFromJson(json);
-    ASSERT_TRUE(result);
+    auto result = Instance::GetListFromJson(json);
+    ASSERT_TRUE(result.has_value());
```

Comment:
ASSERT_THAT(result, HasNoError()); // this has added benefit of automatically printing the error message in the case assert fais.

Reply:
done.